templates/item-info.widget

```diff
@@ -152,64 +152,84 @@ function editItemInfo(itemUid) {
   switchSection("#item-" + itemUid + " .item-info", "editing");
 }
 
+CSS
```

Comment:
This shouldn't be in this PR. 

(You can remove these changes by amending the commit with `templates/item-info.widget` unstaged and force-pushing.)